LeagueToolkit/IO/ReleaseManifestFile/ReleaseManifest.cs

```diff
@@ -61,7 +60,10 @@ public ReleaseManifest(Stream stream)
                     byte[] signature = br.ReadBytes(256);
                     // NOTE: verify signature here
                 }
-                byte[] uncompressedFile = Zstd.Decompress(compressedFile, (int)uncompressedContentSize);
+
+                using var decompressor = new Decompressor();
+                    var uncompressedFile = decompressor.Unwrap(compressedFile).ToArray();
```

Comment:
This indentation (and the one in the two other cases below) is misleading and should instead align with the `using` line.